### PR TITLE
Various fixes in nuspec, projects and packages.config

### PR DESCRIPTION
**Pull request by networkfusion**
## Description
- Fix a number of mismatches in nuspec, project files and packages.config.
- Work on improving the build for the changed bindings. 

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)
- [ ] Unit Tests (work on Unit Tests, has no impact on code or features)
- [ ] Documentation (changes or updates in the documentation, has no impact on code or features)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/main/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by networkfusion** [comment]
I am stuck on this:
image

Seeing if removing from build will help!

**Comment by networkfusion** [comment]
I am not sure it is all fixed (regarding ability to build all projects properly)... but all the checks are now happy...

File: azure-pipelines.yml [hunk 1]
```diff
@@ -265,6 +270,10 @@ steps:
   env:
     MY_GITHUBTOKEN: $(GitHubToken)
 
+# check for app.config files
```
Comment (josesimoes) [inline azure-pipelines.yml:273]: This is a good idea and should be available as a template to allow reuse.

**Comment by josesimoes** [comment]
Azure pipeline and yaml work moved to separate commit: 4aa7c0d0c1ccd71f195933b094c95ce5713183f1.
Rest of the fixes in this PR.

@networkfusion thanks for taking care of these fixes. It was a thorough work! 👍🏻 💯

**Comment by josesimoes** [comment]
@nfbot updatedependencies

**Pull request merged**